server/cmd/mmctl/commands/post.go

```diff
@@ -64,31 +65,56 @@ func init() {
 }
 
 func postCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
-	message, _ := cmd.Flags().GetString("message")
+	message := args[1]
 	if message == "" {
 		return errors.New("message cannot be empty")
 	}
 
-	replyTo, _ := cmd.Flags().GetString("reply-to")
-	if replyTo != "" {
-		replyToPost, _, err := c.GetPost(context.TODO(), replyTo, "")
+	post := &model.Post{
+		Message: message,
+	}
+
+	if strings.HasPrefix(args[0], "@") {
+		userName := strings.Split(args[0], "@")[1]
+		if userName == "" {
+			return errors.New("unable to find user ''")
+		}
+		otherUser, _, err := c.GetUserByUsername(context.TODO(), userName, "")
 		if err != nil {
-			return err
+			return errors.New("unable to find user '" + args[0] + "'")
 		}
-		if replyToPost.RootId != "" {
-			replyTo = replyToPost.RootId
+
+		me, _, err := c.GetMe(context.TODO(), "")
+		if err != nil {
+			return errors.New("unable to get current user")
```

Comment:
Let's wrap the error here and in other places as well
```suggestion
			return errors.Wrap(err, "unable to get current user")
```